cirq/sim/simulator.py

```diff
@@ -25,18 +25,25 @@
 """
 
 from typing import (
-    Any, Dict, Hashable, Iterator, List, Tuple, Union, Optional)
+    Any,
+    Dict,
+    Hashable,
+    Iterator,
+    List,
+    Tuple,
+    Union,
+    Optional,
```

Comment:
nit: might remove final "," since this is a lot of vert space for an import

Reply:
Acknowledged, but I prefer this style (and the formatter accepts it).